### ISSUE TITLE
Traceback error after start

### ISSUE THREAD

**Issue by zdima40** [mhammond/pywin32#1294]
I have error after run PythonWin:
Traceback (most recent call last):
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\framework\intpyapp.py", line 181, in InitInstance
    from . import interact
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\framework\interact.py", line 31, in <module>
    from . import winout
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\framework\winout.py", line 233, in <module>
    import pywin.scintilla.view
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\scintilla\view.py", line 722, in <module>
    LoadConfiguration()
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\scintilla\view.py", line 708, in LoadConfiguration
    configManager = ConfigManager(configName)
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\scintilla\config.py", line 123, in __init__
    line, lineno = self._load_keys(subsection, fp, lineno)
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\scintilla\config.py", line 270, in _load_keys
    sc, flag = keycodes.parse_key_name(key)
  File "D:\Program Files (x86)\Python\Lib\site-packages\pythonwin\pywin\scintilla\keycodes.py", line 91, in parse_key_name
    return vk, flags | this_flags
TypeError: unsupported operand type(s) for |: 'int' and 'NoneType'

Versions:
Python - 3.7.2 (win32)
pywin32 build 224 (pywin32-224.win32-py3.7)
OS - win7

**Comment by zdima40** [comment]
I tried to solve the problem like this: Like helped
```
    #return vk, flags | this_flags
    
    if this_flags is not None:
        flags |= this_flags
    else:
        flags |= 0
        
    return vk, flags
```

**Comment by zdima40** [comment]
I tried to solve the problem like this. Like helped
keycodes.py
```
    #return vk, flags | this_flags
    
    if this_flags is not None:
        flags |= this_flags
    else:
        flags |= 0
        
    return vk, flags
```

**Comment by Aareon** [comment]
This resolved your issue and produced the desired effect?

**Comment by Avasam** [comment]
Fixed by #1264
